editor/src/document/document_message_handler.rs

```diff
@@ -164,8 +164,18 @@ impl MessageHandler<DocumentsMessage, &InputPreprocessor> for DocumentsMessageHa
 					_ => format!("Untitled Document {}", new_doc_title_num),
 				};
 
+				let new_document = match serialized_contents {
+					None => DocumentMessageHandler::with_name(name),
+					Some(content) => {
+						let res = DocumentMessageHandler::with_name_content(name, content);
+						if res.is_err() {
+							// how can we show an error message in the UI?
+							return;
```

Comment:
Is there a standard way of showing the user an error message?

Reply:
Added this for you here: https://github.com/GraphiteEditor/Graphite/commit/89ee35b831e2ddcd3fd196642455f2e93fa353fb